src/test/chain-simulator/accounts.cs-e2e.ts

```diff
@@ -1187,4 +1191,346 @@ describe('Accounts e2e tests with chain simulator', () => {
       expect(response.data).toBeGreaterThanOrEqual(1);
     });
   });
+
+  describe('GET /accounts/:address/transfers', () => {
+    it('should return transfers for a given address', async () => {
+      const countResponse = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers/count`);
+      const expectedCount = countResponse.data;
+
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers`);
```

Comment:
have a check that also the count si > 0. otherwise both endpoints may return no data/count 0

Reply:
done